pkg/util/downloads.go

```diff
@@ -234,7 +234,7 @@ func preamble() (*github.Client, *github.RepositoryRelease, *github.Response, er
 			ts := oauth2.StaticTokenSource(
 				&oauth2.Token{AccessToken: token},
 			)
-			tc = oauth2.NewClient(oauth2.NoContext, ts)
+			tc = oauth2.NewClient(context.TODO(), ts)
```

Comment:
Why this change?

Reply:
NoContext is deprecated: https://godoc.org/golang.org/x/oauth2#pkg-variables